extensions/applicationinsights-angular-js/test-app/tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "experimentalDecorators": true,
     "module": "esnext",
     "moduleResolution": "node",
-    "importHelpers": true,
+    "importHelpers": false,
```

Comment:
This one is probably wrong -- will check and possible revert

Reply:
Kept the removal of tslib, but set the noEmitHelpers to false, so that typescript drops out the __extends() rather than either always using tslib or using the shims modules, as that should only be required for internal AI usage and as long as an __extends() and/or __assign() exists everything should work.